content/docs/en/elements/layouts/grid-layout.md

```diff
@@ -1,34 +1,38 @@
+contributors: [rigor789, ikoevska]
 ---
 
-The GridLayout is a layout that arranges its child elements in a table structure of rows and columns. A cell can contain multiple child elements, they can span over multiple rows and columns, and even overlap each other. The GridLayout has one column and one row by default. To add additional columns and rows, you have to specify column definition items (separated by commas) to the `columns` property and row definition items (separated by commas) to the `rows` property of the GridLayout. The width of a column and the height of a row can be specified as an absolute amount of pixels, as a percentage of the available space or automatically:
+`<GridLayout>` is a layout component that lets you arrange its child elements in a table-like manner. 
 
-- **Absolute**: Fixed size of pixels.
-- **Star (*)**: Takes as much space as available (after filling all auto and fixed sized columns), proportionally divided over all star-sized columns. So `3/7` means the same as `30/70`.
-- **Auto**: Takes as much space as needed by the contained child element(s).
+The grid consists of rows, columns, and cells. A cell can span one or more rows and one or more columns. It can contain multiple child elements which can span over multiple rows and columns, and even overlap each other. 
 
-### Samples
+By default, the `<GridLayout>` has one column and one row. You can add columns and rows by configuring the `columns` and the `rows` property. In these properties, you need to set the number of columns and rows and their width and height. You set the number of columns by listing their widths, separated by a comma. You set the number of rows by listing their heights, separated by a comma.
 
-#### A complex grid layout
+You can set a fixed size for column width and row height or you can create them in a responsive manner. See [Props](#props) for more information.
+
+### Examples
+
+#### Grid layout with fixed sizing
+
+The following example creates a simple 2-by-2 grid with fixed column widths and row heights.
 
 ```html
-<GridLayout columns="40, auto, *" rows="40, auto, *" backgroundColor="#3c495e">
+<GridLayout columns="150, 150" rows="150, 150">
   <Label text="0,0" row="0" col="0" backgroundColor="#43b883"/>
-  <Label text="0,1" row="0" col="1" colSpan="2" backgroundColor="#1c6b48"/>
-  <Label text="1,0" row="1" col="0" rowSpan="2" backgroundColor="#289062"/>
+  <Label text="0,1" row="0" col="1" backgroundColor="#1c6b48"/>
+  <Label text="1,0" row="1" col="0" backgroundColor="#289062"/>
   <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
-  <Label text="1,2" row="1" col="2" backgroundColor="#289062"/>
-  <Label text="2,1" row="2" col="1" backgroundColor="#1c6b48"/>
-  <Label text="2,2" row="2" col="2" backgroundColor="#43b883"/>
 </GridLayout>
 ```
-<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout.svg" />
+
+![Simple GridLayout](/screenshots/simple-grid-layot.jpg)
```

Comment:
You can change it back to `https://art.nativescript-vue.org/layouts/grid_layout.svg` I have updated the graphic in the art repo.